test/chplcheck/MisleadingIndentation.good-fixit

```diff
@@ -20,7 +20,7 @@ writeln(i);
     ("second thing");
 
   // the only fixit here is ignore, don't know the indentation level
-  @chplcheck.ignore("MisleadingIndentation")
+  @chplcheck.ignore("IncorrectIndentation")
```

Comment:
Why is the fixit now `IncorrectIndentation`? I thought we had it set so `IncorrectIndentation` doesn't fire when `MisleadingIndentation` does fire

Reply:
<img width="574" alt="Screen Shot 2024-08-22 at 10 27 45 AM" src="https://github.com/user-attachments/assets/6d2f03fe-ae30-419d-9750-db5bf761a432">

They are separate warnings for separate reasons; one warns about the fact that "second thing" is indented the same as the loop writeln, the other warns about the loop writeln being indented the same as the loop itself. Ideally, a fixit would apply both `ignore` attributes, but we add one at a time only (to avoid mangling the program text), and  don't repeatedly run `chplcheck` to iteratively add all the fixits.

Reply:
Yes they are separate warnings for a reason, but I thought we had logic to avoid this double warning

Reply:
The logic that avoids the double-warning is only there because misleading indentation _is_ incorrect indentation, so the _same_ statement would get warned about twice. In this case, there are two distinct warnings that do not overlap, so I think it's working as intended.

Reply:
Ah I see, the dedup logic is for something like this
```chapel
for i in 1..10 do
 writeln(a);
 writeln(a);
```